content/en/docs/started/workstation/getting-started-linux.md

```diff
@@ -50,7 +50,11 @@ page will complete this path.
 
 ### Kubernetes appliance
 
-Similar to the Kubeflow appliance, the Kubernetes appliance is a virtual machine has a
-Kubernetes cluster already installed. After starting the virtual machine you will need
+A Kubernetes appliance is a virtual machine that has a
+Kubernetes cluster already installed. After starting the virtual machine, you need
 to install Kubeflow. This option gives you full control over your Kubeflow setup.
 
+- Install [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/)
+
+Follow the instructions on [deploying with MiniKube on 
+Linux](/docs/started/workstation/minikube/linux/) to complete this path.
```

Comment:
Thanks @sarahmaddox ! 
Just a minor change: 

The Link should be 
/docs/started/workstation/minikube-linux/

Other option is to change the name of Minikube tutorial.

Reply:
Oh well spotted! Fixed.